src/trading/Orders.sol

```diff
@@ -161,48 +152,46 @@ contract Orders is DelegationTarget, IOrders {
         return _orderId;
     }
 
-    function removeOrder(bytes32 _orderId, Trading.TradeTypes _type, IMarket _market, uint8 _outcome) public onlyWhitelistedCallers returns (bool) {
+    function removeOrder(bytes32 _orderId, Order.TradeTypes _type, IMarket _market, uint8 _outcome) public onlyWhitelistedCallers returns (bool) {
         removeOrderFromList(_orderId, _type, _market, _outcome);
         orders[_orderId].fxpPrice = 0;
         orders[_orderId].fxpAmount = 0;
-        orders[_orderId].owner = 0;
+        orders[_orderId].maker = 0;
         orders[_orderId].fxpMoneyEscrowed = 0;
         orders[_orderId].fxpSharesEscrowed = 0;
         return true;
     }
 
-    function fillOrder(bytes32 _orderId, Trading.TradeTypes _orderType, IMarket _market, uint8 _orderOutcome, uint256 _sharesFilled, uint256 _tokensFilled) public onlyWhitelistedCallers returns (bool) {
+    function fillOrder(bytes32 _orderId, Order.TradeTypes _orderType, IMarket _market, uint8 _orderOutcome, uint256 _sharesFilled, uint256 _tokensFilled) public onlyWhitelistedCallers returns (bool) {
         require(_market.getTypeName() == "Market");
         require(_orderOutcome < _market.getNumberOfOutcomes());
-        require(_orderType == Trading.TradeTypes.Bid || _orderType == Trading.TradeTypes.Ask);
+        require(_orderType == Order.TradeTypes.Bid || _orderType == Order.TradeTypes.Ask);
         require(_orderId != bytes32(0));
         require(_sharesFilled <= orders[_orderId].fxpSharesEscrowed);
         require(_tokensFilled <= orders[_orderId].fxpMoneyEscrowed);
         require(orders[_orderId].fxpPrice <= _market.getMaxDisplayPrice());
         require(orders[_orderId].fxpPrice >= _market.getMinDisplayPrice());
         require(_market.getMaxDisplayPrice() + _market.getMinDisplayPrice() <= 2**254);
         uint256 _fill = 0;
-        if (_orderType == Trading.TradeTypes.Bid) {
+        if (_orderType == Order.TradeTypes.Bid) {
             // We can't use safeSub here because it disallows subtracting negative numbers. Worst case here is an operation of 2**254 - 1 as required above, which won't overflow
-            _fill = _sharesFilled + _tokensFilled.fxpDiv(uint(orders[_orderId].fxpPrice - _market.getMinDisplayPrice()), 10**18);
-        }
-        if (_orderType == Trading.TradeTypes.Ask) {
+            _fill = _sharesFilled + _tokensFilled.fxpDiv(uint(orders[_orderId].fxpPrice - _market.getMinDisplayPrice()), 1 ether);
+        } else if (_orderType == Order.TradeTypes.Ask) {
             // We can't use safeSub here because it disallows subtracting negative numbers. Worst case here is an operation of 2**254 - 1 as required above, which won't overflow
-            _fill = _sharesFilled + _tokensFilled.fxpDiv(uint(_market.getMaxDisplayPrice() - orders[_orderId].fxpPrice), 10**18);
+            _fill = _sharesFilled + _tokensFilled.fxpDiv(uint(_market.getMaxDisplayPrice() - orders[_orderId].fxpPrice), 1 ether);
```

Comment:
I know this was pre-exesting, but you shouldn't need to ever do `fxpDiv(x, 10^18)` as `fxpDiv` is already dividing by 10^18.  We can investigate later if the tests are passing, but something is fishy here.

Reply:
Ok, I'll add a FIXME here to remind us to revisit this.